README.md

```diff
@@ -193,6 +193,21 @@ app.use(function *(){
 app.listen(3000);
 ```
 
+### Using koa@next
+
+```JS
+const errors = require('@google/cloud-errors')();
+const Koa = require('koa');
+const app = new Koa();
+
+// Will attach to 'error' event which includes
+// request/response lifecycle errors
+errors.koaNext(app);
```

Comment:
I think it would be stable for our purposes and the API surface which we wish to exercise. It may break though given that its OSS. These mocks are meant to break, though, given that the API surface changes so that we can know about it